CONTRIBUTING.md

```diff
@@ -1,96 +1,232 @@
+
+A *proposal* becomes a *draft* when the set of problems or drawbacks have been
+fully considered and accepted or resolved, and the solution is deemed
+desirable. A *draft*'s goal is to complete final spec edits that are ready to
+be merged and implement the *draft* in AsyncAPI libraries along with tests to
+gain confidence that the spec text is sufficient.
+
+*Drafts* may continue to evolve and change, occasionally dramatically, and are
+not guaranteed to be accepted. Therefore, it is unwise to rely on a *draft* in a
+production environment. AsyncAPI libraries *should* implement *drafts* to
+provide valuable feedback, though are encouraged not to enable the *draft*
+feature without explicit opt-in when possible.
+
+
+## Stage 3: *Accepted*
+
+An RFC at the *accepted* stage is a completed solution. According to a spec
+Committer it is ready to be merged as-is into the spec document. The RFC is
+ready to be deployed in AsyncAPI libraries. An *accepted* RFC must be
+implemented in AsyncAPI.js.
+
+*Entrance criteria:*
+
+* Consensus the solution is complete
+* Complete spec edits, including examples and prose
+* Compliant implementation in [AsyncAPI JS Parser](https://www.github.com/asyncapi/parser-js) (fully tested and merged or ready to merge)
+
+A *draft* is *accepted* when the Committers have been convinced via
+implementations and tests that it appropriately handles all edge cases; that the
+spec changes not only precisely describe the new syntax and semantics but
+include sufficient motivating prose and examples; and that the RFC includes
+edits to any other affected areas of the spec. Once *accepted*, its *champion*
+should encourage adoption of the RFC by opening issues or pull requests on other
+popular AsyncAPI libraries.
+
+An *accepted* RFC is merged into the AsyncAPI spec's master branch by a Committer
+and will be included in the next released revision.
+
+
+## Stage X: *Rejected*
+
+An RFC may be *rejected* at any point and for any reason. Most rejections occur
+when a *strawman* is proven to be unnecessary, is misaligned with the *guiding
+principles*, or fails to meet the entrance criteria to become a *proposal*.
+A *proposal* may become *rejected* for similar reasons as well as if it fails to
+reach consensus or loses the confidence of its *champion*. Likewise a *draft*
+may encounter unforeseen issues during implementations which cause it to lose
+consensus or the confidence of its *champion*.
+
+RFCs which have lost a *champion* will not be *rejected* immediately, but may
+become *rejected* if they fail to attract a new *champion*.
+
+Once *rejected*, an RFC will typically not be reconsidered. Reconsideration is
+possible if a *champion* believes the original reason for rejection no longer
+applies due to new circumstances or new evidence.
+
 
 ## References
-This document was adapted from the open-source contribution guidelines for [Facebook's Draft](https://github.com/facebook/draft-js/blob/master/CONTRIBUTING.md).
```

Comment:
Hah 🤣

Reply:
😄 No shame. That's open source. They did a great job and so we reuse it.